layouts/partials/header.html

```diff
@@ -15,7 +16,8 @@
                 />
             </a>
         </div>
-        <div class="col-sm-8 col-12 text-sm-left text-center">
+        {{ end }}
+        <div class="col-sm col-12 text-sm-left text-center">
```

Comment:
Do you know if changing `col-sm-8` to `col-sm` makes a difference in styling with/without `home_image` set? I can test this out as well.

Reply:
Nevermind, I was able to test this quickly. Is this the styling you want when `home_image` is omitted from the config on larger screens? It's centered on smaller screens (mobile). I could see the content being centered on larger screens as well.

<img width="862" alt="Screenshot 2023-04-29 at 7 23 59 PM" src="https://user-images.githubusercontent.com/34121980/235332581-80001b8c-e4a4-4a3d-a488-8c42c35162df.png">

Reply:
You're right, centering the header on layouts without a home image looks much better. I adjusted the code accordingly and made the applied CSS classes depend on whether `home_image` is set or not.

The site now looks like this without a home image:

![image](https://user-images.githubusercontent.com/4249096/235480893-f43f09a1-2703-4b81-b6e5-55bcafa92774.png)

Reply:
Looks good to me 🚀